.github/pull_request_template.md

```diff
@@ -1,17 +1,21 @@
 # Description
-<!-- Briefly describe the feature or bug that your pull request addresses, 1-2 sentences. -->
 
+<!-- Briefly describe the feature or bug that your pull request addresses, 1-2 sentences. -->
```

Comment:
This was totally unrelated, but I fixed my linter, and it insisted on "fixing" this file.